experiments/mountain_car_plots.py

```diff
@@ -0,0 +1,269 @@
+"""Heatmaps and reward plotting code for debugging MountainCar."""
```

Comment:
Can you call out which functions are useful as an "entry point"? It seems a lot of them are helper methods.

You could either make the helper methods start with `_`, or just document which methods people should be looking at in the module docstring.

Reply:
I either moved helper methods out of the file (into `.rewards.common`), or prepended `_` to make functions private. Every public function here is now a potential Figure-generating entry point for analyzing mountain car rewards.